lms/services/lti_user.py

```diff
@@ -75,6 +78,7 @@ def deserialize(self, **kwargs: dict) -> LTIUser:
         lti = LTI(
             course_id=lti_data["course_id"],
             assignment_id=lti_data["assignment_id"],
+            product_family=lti_data["product_family"],
```

Comment:
And when building a LTIUser form a serialized one, just use the string available there.